### ISSUE TITLE
amp lightbox animate-in attribute removed by sanitizer 

### ISSUE THREAD

**Issue by p2er** [Automattic/amp-wp#1330]
**Description**
Optional attribute animate-in is being removed from amp-lightbox and therefore does not allow animation configuration for lightbox even though included in the official documentation: https://www.ampproject.org/docs/reference/components/amp-lightbox#animate-in-(optional)

**How to demo**
- Add lightbox to page template
- Create page
- Open AMP version of page
- **IS:** animate-in attribute has been removed
- **SHOULD:** still have animate-in attribute in page MARKUP

**Example**
```
<button on="tap:demo-lb.open">open</button>
<amp-lightbox id="demo-lb" layout="nodisplay" scrollable animate-in="fly-in-top">
<button on="tap:demo-lb.close">close</button>
<p>This lightbox should animate into viewport from top.</p>
</amp-lightbox>
```

**Comment by hellofromtonya** [comment]
@p2er I can replicate this issue.  

We'll need to update the `'attr_spec_list` for `amp-lightbox` in `AMP_Allowed_Tags_Generated`

https://github.com/Automattic/amp-wp/blob/677b7617ffb47094dab04f76e91dafb57e67ad0c/includes/sanitizers/class-amp-allowed-tags-generated.php#L2264-L2286

While in there, we should also check the attributes to [the spec](https://www.ampproject.org/docs/reference/components/amp-image-lightbox) for `amp-image-lightbox`.

**Comment by westonruter** [comment]
This will be resolved by #1315. We just need to finish it out.

**Comment by kienstra** [comment]
**Moving To 'Ready For Merging'**

If it's alright, I'm moving this to 'Ready For Merging,' as I don't think it needs functional testing. Feel free to move it back.
